hivemind/p2p/p2p_daemon.py

```diff
@@ -73,6 +73,20 @@ async def create(cls, *args, quic=1, tls=1, conn_manager=1, dht_client=1,
             break
         return self
 
+    @classmethod
+    async def replica(cls, daemon_listen_port: int, host_port: int):
```

Comment:
Suggestion: rename this method into a verb (replicate) or context (from_running_daemon)?